martian: invalid error printing

2024/01/29 11:50:11.339903 [proxy] [ERROR] [97023-1de7875a] CONNECT tunnel: %!w(*fmt.wrapError=&{got error while flushing response back to client: write tcp IP:PORT->IP:PORT: write: connection reset by peer 0xc00221d310})

--- a/internal/martian/proxy_handler.go
+++ b/internal/martian/proxy_handler.go
@@ -181,7 +181,7 @@ func (p proxyHandler) handleUpgradeResponse(rw http.ResponseWriter, req *http.Re
 	res.Body = nil
 
 	if err := p.tunnel(resUpType, rw, req, res, uconn); err != nil {
-		log.Errorf(ctx, "%s tunnel: %w", resUpType, err)
+		log.Errorf(ctx, "%s tunnel: %v", resUpType, err)
 		panic(http.ErrAbortHandler)
 	}
 }

